src/components/MentorsList/MentorList.css

```diff
@@ -12,22 +8,27 @@
     transform: translateY(0);
     transition: transform 0.3s ease;
   }
-
   .mentors-wrapper.active {
     transform: translateY(190px);
     margin-bottom: 50px;
   }
 }
 
-.mentors-wrapper > .mentors-cards {
-  display: flex;
-  flex-wrap: wrap;
-  justify-content: center;
-  overflow: auto;
-  margin-top: 30px;
+@media all and (min-width: 580px) {
+  .mentors-wrapper {
+    padding-bottom: 30px;
+    margin-left: 276px;
+  }
+  .mentors-wrapper>.mentors-cards {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    overflow: auto;
```

Comment:
Can we also set this just to `overflow-y: auto` so we don't get a horizontal scrollbar?

Reply:
I'm never getting a horizontal scrollbar. But, this change also keeps from getting a horizontal scrollbar - so, changed.